editoast/src/views/pagination.rs

```diff
@@ -65,16 +66,49 @@ fn default_page() -> i64 {
     1
 }
 
+impl PaginationQueryParam {
+    pub fn validate(&self, max_page_size: i64) -> Result<PaginationQueryParam> {
+        let page_size = self.page_size.unwrap_or(25).max(10);
```

Comment:
```suggestion
        let page_size = self.page_size.unwrap_or(25);
```

You won't trigger `page_size < 1` otherwise